Makefile

```diff
@@ -22,7 +22,11 @@ docs:
 
 check-docs:
 	@for m in $(MODULES); do \
+		echo $$m; \
 		cd $$m; \
 		../scripts/update-readme.sh check || exit $$?; \
 		cd ..; \
-	done;
\ No newline at end of file
+	done;
+
+test: fmt
+	GOCACHE=off AWS_PROFILE=czi-si go test ./...
```

Comment:
why `GOCACHE=off`?

Reply:
go wil cache test results. cache is keyed off of go code. so if you change terraform but not go, it will not re-run the tests.